app/code/local/Aoe/FraudManager/Model/Rule/Condition/Order/Item/Attribute.php

```diff
@@ -4,6 +4,7 @@ class Aoe_FraudManager_Model_Rule_Condition_Order_Item_Attribute extends Aoe_Fra
 {
     protected $attributes = [
         'sku' => ['SKU', ['==', '!=', '{}', '!{}', 'RE']],
+        'original_price' => ['Original Price', ['==', '!=', '{}', '!{}', 'RE']],
```

Comment:
I think you need to be using some different operators here. You are missing '>', '<', '>=', '<='. Also, RE doesn't make sense for a number check.